app/src/main/java/edu/mills/cs115/AZCategoryFragment.java

```diff
@@ -1,11 +1,9 @@
 package edu.mills.cs115;
 
```

Comment:
Will you leave these lines blank in order to organize/categorize the import statements?